iFixit/react-commerce
# Add carousel to product page

closes #674 
closes #669 

This PR introduces a new product gallery component to display product images.
The product page contains two product galleries: one for desktop, one for mobile.
The two galleries are kept in sync so that the same image is shown even if the screen dimensions changes (eg rotating a mobile device).
The whole productSection inside the product page (so gallery + product variant selectors) is now responsive

## QA

1. Visit the Vercel preview
2. Check that the gallery is working properly
3. Check that mobile and desktop galleries are kept in sync
4. Check that the product section is responsive
